<?xml version="1.0"?>
<launch>

  <node pkg="region_growing_segmentation" type="region_growing_segmentation_node" name="region_growing_segmentation_node" output="screen"
        >
<!--
    <param name="SavePath" value="/home/xjh/toxjh/"/>
    <param name="TopicName" value="/course_agv/laser/scan"/>
    <param name="FilterSize" value="0.005"/>
    <param name="AngleMin" value="-3.2"/>
    <param name="AngleMax" value="3.2"/>
    launch-prefix="xterm -e gdb -ex run- -args"
-->
    <param name="NDT_TransformationEpsilon" value="0.001"/>
    <param name="NDT_StepSize" value="0.1"/>
    <param name="NDT_Resolution" value="1"/>
    <param name="NDT_MaximumIterations" value="30"/>
    <param name="NDT_OulierRatio" value="0.2"/>
<!--
    <param name="Mapping_KeyFrameDistance" value="0.4"/>
    <param name="Mapping_NumKeyFrameInLocalMap" value="100"/>
    <param name="Mapping_TargetCloudResolution" value="0.01"/>
    <param name="Mapping_SourceCloudResolution" value="0.005"/>
-->
  </node>

  <node pkg="rviz" type="rviz" name="rviz"
    args="-d $(find region_growing_segmentation)/rviz/default.rviz" />

</launch>
